atlp-rwanda/atlp-pulse-mobile
# #103 Full Screen Sidebar Implementation

### What does this PR do?
Make our sidebar a full-screen drawer.
Profile Sidebar for Quick Access

### How should this be manually tested?
To test this you need to clone this project and switch to `ft-fullscreen-sidebar-implementation` branch, install all dependencies `npm install` then start the project `npm start`.
after successfull logging in click on profile icon then choose check if full screen sidebar is being opened with three links on it(profile, settings and login activity).

Click also on menu icon and see if sidebar  fit the screen(width)

### Screenshots

<img width="180" src="https://github.com/user-attachments/assets/5cf9fd05-d368-4cb5-bb47-5a9970d72c21" />
<img width="180" src="https://github.com/user-attachments/assets/2e14cf69-93f4-416a-8d7d-a854242cc5d7" />
<img width="180" src="https://github.com/user-attachments/assets/a52d44ae-8dbe-4767-90d9-0551b6f25323" />
<img width="180" src="https://github.com/user-attachments/assets/16c0b4cb-217a-4504-9995-0b79f3c432cb" />
